constitution.md

```diff
@@ -1,299 +1,262 @@
-
-       **Non-Profit Clause**
-
-       The assets and income of the Club/Society shall be applied solely in furtherance of its above- mentioned objects and no portion shall be distributed directly or indirectly to the members of the Club/Society except as bona fide compensation for services rendered or expenses incurred on behalf of the Club/Society.
-
-       **Notes**
-
-       The Club/Society shall not adopt aims or objectives, which would bring it under the jurisdiction of the University of Queensland Sports Association (UQ Sport).
-
-## INTERPRETATION
-1. In this Constitution, unless the contrary intention appears:
-
-    1. “Semester” refers to Semester 1 or 2 as prescribed by the University's Academic Calendar
+## University of Queensland Computing Society
 
-## MEMBERSHIP
-1. Membership is open to:
-
-    1. Any student of the University of Queensland
-
-    2. Any University of Queensland Academic Staff Member
-
-    3. any other members of the University or wider community interested in Club/Society's activities
-
-       **Notes**
-
-       The Club/Society shall have not less than:
-       * Thirty (30) financial members in the case of a subsidised club
-       * Fifteen (15) financial members in the case of a publications club
-       * Ten (10) financial members in the case of an Ipswich based club.
-       * 70% of the Club/Society's membership must be students.
-
-1. Membership Fees
-
-    1. The membership fees shall be such sum, as the members and/or management committee shall from time to time so determine, but shall not be less than the minimum amount set by the University of Queensland Union.
-
-    2. The membership fees shall be payable upon joining the Club/Society.
-
-1. Termination of Membership
-
-    1. A member may resign from the Club/Society at any time by giving notice in writing to the secretary.
-
-    2. Such resignation shall take effect at the time such notice is received by the secretary unless a later date is specified in the notice when it shall take effect on that later date.
-
-    3. If a member -
-
-        1. fails to comply with any of the provisions of these rules
-
-        2. has membership fees in arrears
-
-        3. conducts himself or herself in a manner considered to be injurious or prejudicial to the character or interests of the Club/Society, the management committee shall consider whether the member's membership shall be terminated.
+**Constitution - Affiliated Bodies**
+**Version 1.4**
+**30th of May 2019**
 
-    4. The member concerned shall be given a full and fair opportunity of presenting the member's case and if the management committee resolves to terminate the membership it shall instruct the secretary to advise the member in writing accordingly.
+### 1 Preliminaries
 
-1. Register of Members
+1.1 The Society shall be called “University of Queensland Computing Society”, hereafter referred to as “the Society”  
```

Comment:
```suggestion
1.1 The Society shall be called “University of Queensland Computing Society”  
```

Reply:
Is this material enough to not be considered formatting?

Reply:
Yes.

Reply:
Aight, I've removed it from this formatting PR.